<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Core::Internal::GeneralSettings</class>
 <widget class="QWidget" name="Core::Internal::GeneralSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>295</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="interfaceBox">
     <property name="title">
      <string>User Interface</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="colorLabel">
        <property name="text">
         <string>Color:</string>
        </property>
        <property name="buddy">
         <cstring>colorButton</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="languageBox">
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
          </property>
          <property name="minimumContentsLength">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>285</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="Utils::QtColorButton" name="colorButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>0</height>
           </size>
          </property>
          <property name="alphaAllowed" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="resetButton">
          <property name="toolTip">
           <string>Reset to default</string>
          </property>
          <property name="text">
           <string>R</string>
          </property>
          <property name="icon">
           <iconset resource="core.qrc">
            <normaloff>:/core/images/reset.png</normaloff>:/core/images/reset.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>285</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="languageLabel">
        <property name="text">
         <string>Language:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="systemBox">
     <property name="title">
      <string>System</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="terminalLabel">
        <property name="text">
         <string>Terminal:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="terminalEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="resetTerminalButton">
        <property name="toolTip">
         <string>Reset to default</string>
        </property>
        <property name="text">
         <string>R</string>
        </property>
        <property name="icon">
         <iconset resource="core.qrc">
          <normaloff>:/core/images/reset.png</normaloff>:/core/images/reset.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="externalFileBrowserLabel">
        <property name="text">
         <string>External file browser:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="externalFileBrowserEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="resetFileBrowserButton">
        <property name="toolTip">
         <string>Reset to default</string>
        </property>
        <property name="text">
         <string>R</string>
        </property>
        <property name="icon">
         <iconset resource="core.qrc">
          <normaloff>:/core/images/reset.png</normaloff>:/core/images/reset.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QToolButton" name="helpExternalFileBrowserButton">
        <property name="text">
         <string>?</string>
        </property>
        <property name="icon">
         <iconset resource="core.qrc">
          <normaloff>:/core/images/help.png</normaloff>:/core/images/help.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="modifiedLabel">
          <property name="text">
           <string>When files are externally modified:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="reloadBehavior">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>Always Ask</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Reload All Unchanged Editors</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ignore Modifications</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::QtColorButton</class>
   <extends>QToolButton</extends>
   <header location="global">utils/qtcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>colorButton</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>languageBox</tabstop>
  <tabstop>terminalEdit</tabstop>
  <tabstop>resetTerminalButton</tabstop>
  <tabstop>externalFileBrowserEdit</tabstop>
  <tabstop>resetFileBrowserButton</tabstop>
  <tabstop>helpExternalFileBrowserButton</tabstop>
  <tabstop>reloadBehavior</tabstop>
 </tabstops>
 <resources>
  <include location="core.qrc"/>
 </resources>
 <connections/>
</ui>
